src/loops/simple_programs_with_repitition/easy/q5/WordCountTest.java

```diff
@@ -0,0 +1,35 @@
+package loops.simple_programs_with_repitition.easy.q5;
+
+import global.variables.*;
+import global.variables.clauses.*;
+import global.BaseTest;
+
+public class WordCountTest extends BaseTest {
+
+    String input = "This is just for testing. Do not copy!";
+
+    public Clause[] testSentence() {
+
+        Clause[] output = new Clause[3];
```

Comment:
Let's return the clause array the normal way unless we need to do it with indices